controller/src/main/java/io/pravega/controller/server/rest/RESTServer.java

```diff
@@ -81,9 +83,9 @@ protected void shutDown() throws Exception {
         long traceId = LoggerHelpers.traceEnterWithContext(log, this.objectId, "shutDown");
         try {
             log.info("Stopping REST server listening on port: {}", this.restServerConfig.getPort());
-            channel.close();
+            final GrizzlyFuture<HttpServer> shutdown = httpServer.shutdown(30, TimeUnit.SECONDS);
```

Comment:
Q: As per the below example there is a shutdownNow() api too, any particular reason for usng the above?

`https://github.com/jersey/jersey/blob/master/examples/server-async/src/main/java/org/glassfish/jersey/examples/server/async/App.java`